src/main/scala/org/camunda/feel/api/FeelEngineBuilder.scala

```diff
@@ -82,5 +78,18 @@ case class FeelEngineBuilder private (
 
 object FeelEngineBuilder {
 
+  /** Creates a new builder for the FEEL engine.
+    *
+    * @return
+    *   a new builder
+    */
   def create(): FeelEngineBuilder = FeelEngineBuilder()
+
+  /** Creates a new preconfigured builder for the FEEL engine. Use it if the engine is called from
+    * Java code.
+    *
+    * @return
+    *   a new builder
+    */
+  def forJava(): FeelEngineBuilder = FeelEngineBuilder().withCustomValueMapper(new JavaValueMapper)
```

Comment:
❓ As a Scala newbie, this is kind of a cleanup to not make the utility method part of the actual builder interface but expose it on the singleton object directly?

Reply:
Yes. :+1: